db/seeds.dev.rb

```diff
@@ -89,8 +89,7 @@ def create_policy(account, canonical_profile, hosts)
 
     policy = create_policy(account, canonical_profile, hosts)
 
-    passing_hosts_count = ENV['PASSING_HOST_COUNT'].to_i
-    passing_hosts_count ||= rand(1..policy.hosts.count / 2)
+    passing_hosts_count = (ENV['PASSING_HOST_COUNT'] || rand(1..policy.hosts.count / 2)).to_i
```

Comment:
Looks like this is the only line that needs to change. Other changed lines have had Integer.

Reply:
The problem wasn't having non-integers, but converting `nil` to `0` all the time, which rendered the `||` operators useless and so generated 0 entities.